### PR TITLE
Add custom admonitions for right and wrong examples

### PR DESCRIPTION
**Pull request by orlade-anz**
See https://v2.docusaurus.io/docs/markdown-features/#calloutsadmonitions and https://github.com/elviswolcott/remark-admonitions#custom-types for details.

Usage:
:::right {optional custom title}
{markdown}
:::

:::wrong {optional custom title}
{markdown}
:::

Screen Shot 2020-06-25 at 6 00 08 pm

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -43,8 +43,22 @@ module.exports = {
...
+          admonitions: {
+            infima: true,
+            customTypes: {
+              right: {
+                ifmClass: 'success',
+                keyword: 'right',
+                svg: '<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M20.285 2l-11.285 11.567-5.286-5.011-3.714 3.716 9 8.728 15-15.285z"/></svg>',
+              },
+              wrong: {
+                ifmClass: 'danger',
```
Comment (cuminandpaprika) [inline docusaurus.config.js:56]: A little bit confused by the docs here: 

> The default keywords are important, tip, note, warning, and danger. Aliases for info => important, success => tip, secondary => note and danger => warning have been added for Infima compatibility.

from https://github.com/elviswolcott/remark-admonitions#custom-types

Does this mean that valid values for ifmClass are `important | tip | note | warning | danger`?

Specifically I'm confused as to what the ifmClass `danger` resolves to: warning or danger?

Comment (orlade-anz) [reply docusaurus.config.js:56]: I think basically both `danger` and `warning` = red. This makes it generate a div with class `alert alert--danger` which is red.

Comment (cuminandpaprika) [reply docusaurus.config.js:56]: Awesome, both work then, great!

**Comment by ChloePlanet** [comment]
Merged in #13

**Pull request closed without merging**